### DISCUSSION TITLE
How to disable animation for switching tabs

### DISCUSSION THREAD

**Discussion by hholst80** [neovide/neovide#2392] [Q&A] [answered]
Hi neovide community!

In a tab based workflow I find the cursor + text scroll animation confusing and I would like to turn it off.
 
My use case: If I have two separate files open with similar content, the animation that scrolls the cursor to the previous location creates an "optical illusion" that the I am looking in the same file but in a different place. It would be more clear that it is a new file if it did not animate when switching between tabs and just instantly changed the contents of the screen.

Is there a way to customize neovide to this behavior?

Best,
Henrik

**Comment by fredizzimo** [comment, marked as answer]
This is a bug 
* https://github.com/neovide/neovide/issues/1771

But it needs to be fixed in Neovim, not Neovide.
